openfisca_aotearoa/variables/acts/social_security/accommodation_supplement/accommodation_supplement.py

```diff
@@ -72,12 +80,406 @@ class accommodation_supplement__below_income_threshold(Variable):
+        base_rate = people("accommodation_supplement__base", last_week)
+
+        ssa_sched_4_part_7_1 = (
+            + (situation == AccommodationSupplement__Situation.situation_1)
+            * (
+                + accommodation_costs
+                - (
+                    + rebate["section_1"]["accommodation_costs"]
+                    * (accommodation_costs - rebate["section_1"]["base_rate"] * base_rate)
+                    )
+                )
+            )
+
+        ssa_sched_4_part_7_2 = (
+            + (situation == AccommodationSupplement__Situation.situation_2)
+            * (
+                + accommodation_costs
+                - (
+                    + rebate["section_2"]["accommodation_costs"]
+                    * (accommodation_costs - rebate["section_2"]["base_rate"] * base_rate)
+                    )
+                )
+            )
+
+        ssa_sched_4_part_7_3 = (
+            + (situation == AccommodationSupplement__Situation.situation_3)
+            * (
+                + accommodation_costs
+                - (
+                    + rebate["section_3"]["accommodation_costs"]
+                    * (accommodation_costs - rebate["section_3"]["base_rate"] * base_rate)
+                    )
+                )
+            )
+
+        ssa_sched_4_part_7_4 = (
+            + (situation == AccommodationSupplement__Situation.situation_4)
+            * (
+                + accommodation_costs
+                - (
+                    + rebate["section_4"]["accommodation_costs"]
+                    * (accommodation_costs - rebate["section_4"]["base_rate"] * base_rate)
+                    )
+                )
+            )
+
+        ssa_sched_4_part_7_5 = (
+            + (situation == AccommodationSupplement__Situation.situation_5)
+            * (
+                + accommodation_costs
+                - (
+                    + rebate["section_5"]["accommodation_costs"]
+                    * (accommodation_costs - rebate["section_5"]["base_rate"] * base_rate)
+                    )
+                )
+            )
+
+        ssa_sched_4_part_7_6 = (
+            + (situation == AccommodationSupplement__Situation.situation_6)
+            * (
+                + accommodation_costs
+                - (
+                    + rebate["section_6"]["accommodation_costs"]
+                    * (accommodation_costs - rebate["section_6"]["base_rate"] * base_rate)
+                    )
+                )
+            )
+
+        return (
+            + ssa_sched_4_part_7_1
+            + ssa_sched_4_part_7_2
+            + ssa_sched_4_part_7_3
+            + ssa_sched_4_part_7_4
+            + ssa_sched_4_part_7_5
+            + ssa_sched_4_part_7_6
+            )
+
+
+class accommodation_supplement__part_of_nz(Variable):
+    label = "TODO"
+    reference = "https://datafinder.stats.govt.nz/layer/27780-urban-area-2017-generalised-version/"
```

Comment:
https://www.workandincome.govt.nz/map/deskfile/extra-help-information/accommodation-supplement-tables/definitions-of-areas.html

Reply:
We'll let front take care of any transformation.